spec/features/admin_can_log_in_to_dashboard_spec.rb

```diff
@@ -18,7 +18,7 @@
     end
 
     it 'cannot access admin dashboard as registered user' do
-      user = User.create(username: 'rossedfort', password: 'password', role: 0)
+      user = User.create(username: 'rossedfort', password: 'password', first_name: 'Ross', last_name: 'Edfort', email_address: 'rossedfort@yahoo.com', phone_number: '908-698-9024', role: 0)
```

Comment:
Useless assignment to variable - `user`.<br>Line is too long. [189/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.